utils/releaser/src/ReleaseWorker/Release/CreateAndCommitLockFilesReleaseWorker.php

```diff
@@ -33,23 +33,29 @@ public function getPriority(): int
      */
     public function work(Version $version): void
     {
-        $this->symfonyStyle->note('Removing vendor/, node_modules/, composer.lock, and package-lock.json');
+        $this->symfonyStyle->note('Removing vendor/, node_modules/, composer.lock, migrations-lock.yml, and package-lock.json');
         $this->processRunner->run('rm -rf project-base/vendor');
         $this->processRunner->run('rm -rf project-base/node_modules');
         $this->processRunner->run('rm -f project-base/composer.lock');
         $this->processRunner->run('rm -f project-base/package-lock.json');
+        $this->processRunner->run('rm -f project-base/migrations-lock.yml');
         $this->processRunner->run('cd project-base && composer install && npm install');
 
         $this->processRunner->run('git add -f project-base/composer.lock');
         $this->processRunner->run('git add -f project-base/package-lock.json');
+
+        $this->processRunner->run('rm -rf project-base/vendor');
+        $this->processRunner->run('composer install');
```

Comment:
This composer install is not necessary

Reply:
It was added to ensure that application is still working after this step (that calling composer install in project base didn't brake anything)